foruma-frontend/src/views/Login.vue

```diff
@@ -1,20 +1,45 @@
 <template>
-  <LoginForm @submit="onSubmit" />
+  <div>
+    <h1>{{ this.message }}</h1>
+    <LoginForm @submit="onSubmit" />
+  </div>
 </template>
 
 <script lang="ts">
 import { defineComponent } from "vue";
 import LoginForm from "../stories/LoginForm.vue";
 import type { Submit as LoginFormSubmit } from "../stories/LoginForm.vue";
+import axios from "axios";
 
 export default defineComponent({
   name: "Login",
   components: {
     LoginForm,
   },
+  data: function() {
+    return {
+      message: ''
+    }
+  },
   methods: {
-    onSubmit(value: LoginFormSubmit) {
-      console.log(value);
+    async onSubmit(value: LoginFormSubmit) {
+      const response = await axios.post(
+        `${process.env.VUE_APP_API_BASE_URL}/login`,
+        {
+          username: value.username,
+          password: value.password,
+        },
+        {
+          validateStatus: null
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/agabani/foruma/pullRequest?prid=7413717)
```suggestion
          validateStatus: null,
```